AppCenter/AppCenter/Internals/Model/MSAbstractLogPrivate.h

```diff
@@ -1,3 +1,19 @@
+#import <Foundation/Foundation.h>
+
+static NSString *const kMSDevice = @"device";
+static NSString *const kMSDistributionGroupId = @"distributionGroupId";
+static NSString *const kMSSId = @"sid";
+static NSString *const kMSType = @"type";
+static NSString *const kMSTimestamp = @"timestamp";
+static NSString *const kMSUserId = @"userId";
+
+static NSString *const kMSBooleanTypedPropertyType = @"boolean";
```

Comment:
Do we need those? It is redundant and all property types are defined in each header.